<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageBoxDlg</class>
 <widget class="QDialog" name="MessageBoxDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget#widget
{
	background-color: rgb(135, 150, 199);;
	font-size:18px;
	color: rgb(70, 79, 97);
	font-family:Microsoft Yahei;
}
QDateTimeEdit
{
	border-color: rgb(63, 63, 63);

}
QLabel
{
  	font-size:18px;
	color: rgb(63, 63, 63);
	font-family:Microsoft Yahei;
}
QPushButton
{
	background-color: rgb(155, 167, 179);
	font:bold;
	border-color: rgb(176, 186, 221);
	border-width: 2px;
    border-style: solid;
border-radius:5px;
}

QPushButton:hover
{
   border-color: rgb(255, 255, 255);
}
QPushButton:pressed
{
   border-color: rgb(255, 255, 255);
}



QScrollBar:vertical
{
    width:17px;
    background:rgba(0,0,0,0%);
    margin:0px,0px,0px,0px;
    padding-top:17px;
    padding-bottom:17px;
}

QScrollBar::handle:vertical
{
    width:17px;
    background:rgba(0,0,0,25%);
    border-radius:4px;
    min-height:20;
}

QScrollBar::handle:vertical:hover
{
    width:17px;
    background:rgba(0,0,0,50%);
    border-radius:4px;
    min-height:20;
}

QScrollBar::add-line:vertical
{
    height:17px;width:17px;
    border-image:url(:/icon/res/scroll_down_normal.png);
    subcontrol-position:bottom;
}

QScrollBar::sub-line:vertical
{
    height:17px;width:17px;
    border-image:url(:/icon/res/scroll_up_normal.png);
    subcontrol-position:top;
}
QScrollBar::add-line:vertical:hover
{
	height:17px;width:17px;
    border-image:url(:/icon/res/scroll_down_click.png);
    subcontrol-position:bottom;
}
QScrollBar::sub-line:vertical:hover
{
	height:17px;width:17px;
    border-image:url(:/icon/res/scroll_up_click.png);
    subcontrol-position:top;
}
QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical
{
    background:rgba(0,0,0,10%);
    border-radius:4px;
}

QTreeView::item
{
	margin:3px;
}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>248</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{border-radius:10px;
	color:white;
	}
</string>
   </property>
   <widget class="QPushButton" name="btnOK">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>110</y>
      <width>83</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	color: rgb(255, 255, 255);
	font: 75 14pt &quot;Microsoft Yahei&quot;;
}
QPushButton{border-image:url(:/icon/res/button-normal.png);}
QPushButton:hover{border-image:url(:/icon/res/button-move.png);}
QPushButton:pressed{border-image:url(:/icon/res/button-click.png);}</string>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>1</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Yahei</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>提示</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblMessage">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>38</y>
      <width>160</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Yahei</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>提示</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>53</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#label{background-image: url(:/icon/res/msg_symbol.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>110</y>
      <width>83</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	color: rgb(255, 255, 255);
	font: 75 14pt &quot;Microsoft Yahei&quot;;
}
QPushButton{border-image:url(:/icon/res/button-normal.png);}
QPushButton:hover{border-image:url(:/icon/res/button-move.png);}
QPushButton:pressed{border-image:url(:/icon/res/button-click.png);}</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
